src/main/resources/META-INF/plugin.xml

```diff
@@ -158,6 +164,12 @@
                 <override-text place="GitMacheteToolbar"/>
             </action>
 
+            <action id="GitMachete.MergeParentIntoCurrentBranchAction"
+                    class="com.virtuslab.gitmachete.frontend.actions.toolbar.MergeParentIntoCurrentBranchAction"
+                    icon="MacheteIcons.MACHETE_MERGE_PARENT">
+                <override-text place="GitMacheteToolbar"/>
```

Comment:
Actually, to avoid overloading the toolbar with icons... would it be doable to have an option list next to the icon?
Similarly to what's in Git tool window -> Local Changes tab -> Group by action... although arguably the problem there is that with a widget like that the users won't be able to rebase/merge to parent with single click :/

Reply:
hmm, when I use Git Machete I perform the branch actions from the context menu - it makes me feel like I point precisely on which branch I want to do smth. Going forward with such thinking we could leave discover, toggle, edit refresh and fetch Xd BUT it's only my perspective 

And here is why I want to bring https://github.com/VirtusLab/git-machete-intellij-plugin/issues/270 up 😆

Reply:
Huh #270 ... I also thought of that as well, although that would be pretty well hidden from the sight of an average user 🤔

Reply:
we can mention it in the help dialog

Reply:
Okay let's for now leave as is, with merge-to-parent button present by default